frontend/components/sidebar/left/SidebarLeft.vue

```diff
@@ -4,10 +4,20 @@
       sidebar.collapsed = false;
       setContentScrollable();
     "
+    @focus="
+      sidebar.collapsed = false;
+      setContentScrollable();
+    "
     @mouseleave="
       sidebar.collapsed = true;
       setContentScrollable();
     "
+    @blur="
+      sidebar.collapsed = true;
+      setContentScrollable();
+    "
+    role="menu"
```

Comment:
Minor note here: as we are switching from the activist logo to an icon when we switch to a collapsed sidebar we are losing focus on blur when we tab from the sidebar itself to the logo :) Do you want to send along a quick fix for this?

Note that I have a commit coming with some changes shortly 😊

Reply:
As this is sidebar related, maybe we can add this into #338 so we can merge this in, @lucianmurmurache :) :)